deployment/terraform/playgrounds.tf

```diff
@@ -0,0 +1,139 @@
+# Security Group
+resource "aws_security_group" "playgrounds" {
+  vpc_id = aws_vpc.plutus.id
+  name   = "${local.project}_${var.env}_playgrounds"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "TCP"
+    cidr_blocks = concat(var.public_subnet_cidrs, var.private_subnet_cidrs)
+  }
+
+  ## inbound (world): http
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "TCP"
+    cidr_blocks = concat(var.public_subnet_cidrs, var.private_subnet_cidrs)
+  }
+
+  # prometheus node exporter
+  ingress {
+    from_port   = local.node_exporter_port
+    to_port     = local.node_exporter_port
+    protocol    = "TCP"
+    cidr_blocks = var.private_subnet_cidrs
+  }
+
```

Comment:
please add a comment

Reply:
I've added named ports now so it's obvious what they are